plugin-cardea/README.md

```diff
@@ -0,0 +1,15 @@
+# Cardea
+[Cardea](https://github.com/oicr-gsi/cardea) is an API server that serves GSI QC ETL data.
```

Comment:
I wasn't around for cardea's design, but i thought it serves QC Gate ETL data?

Reply:
You are entirely correct. Fixed!